bin/phpunit-docker.sh

```diff
@@ -62,6 +62,12 @@ trap cleanup EXIT
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 WP_VERSION=$($DIR/download-wp-tests.sh wordpress_test root "$MYSQL_ROOT_PASSWORD" "$DB_CONTAINER_NAME" "$WP_VERSION")
 
+if [ -f vendor/bin/phpunit ]; then
+	ENTRYPOINT="--entrypoint /app/vendor/bin/phpunit"
+else
+	ENTRYPOINT=""
+fi
```

Comment:
This adds support for WordPress Nightly. The problem with the current configuration is that if you are using the "global" phpunit (in our case, the one provided by the `ghcr.io/automattic/phpunit-docker/phpunit`), the polyfills need to be installed globally as well.

Since we have installed `yoast/phpunit-polyfills` locally, we have to override the entry point script for the container.
`docker history` says that the original entry point was `/usr/local/bin/phpunit`, therefore we can painlessly update it to `/app/vendor/bin/phpunit` provided that the file exists.